cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -162,6 +162,25 @@ data:
   #     key: node-role.kubernetes.io/etcd
   #     operator: Exists
 
+  # (Optional) CEPH CSI provisioner resource requirement list, Put here list of resources requests and
+  # limits you want to apply for provisioner pod in YAML format
+  # CSI_PROVISIONER_RESOURCE_REQUIREMENT: |
+  #   - requests:
+  #      memory: "512Mi"
```

Comment:
Are these the recommended requests/limits for the provisioner and plugins? These seem rather high. Does the CSI driver really take this much memory/cpu? Even when there are many requests I would think the usage would be lower.

Reply:
Its not clear. I've seen it crash with even more requested. It greatly depends I think if the driver is fuse/nbd or native, as the former requires its own resources.

Reply:
No these are example values, the user is expected to update these values based on the requirement( as this is advance configuration).